# Why is there a key error?

d = {"Name": "John", "Surname": "Smith"}
print(d["Smith"])
